docs/developer-guide-policy.md

```diff
@@ -0,0 +1,9 @@
+# NodeNetworkConfigurationPolicy controller
+
+This controller runs on every node, when it reconciles `NodeNetworkConfigurationPolicy`,
+it checks whether it matches its node. When no `nodeSelector` is specified in the
+object, it always matches. Otherwise it compares the selector with node's labels.
+
+Configuration specified in `desiredState` is then merged with all the other
```

Comment:
We don't have this still, also we have to see what happends now, if more than one is applied

What I have discover also is that nmstate kind of merge it for us, since it applies desired state one after another and not present interfaces are not deleted,

Reply:
Neat! Sure, will drop it from the doc.